pulp_container/app/registry_api.py

```diff
@@ -947,8 +914,10 @@ def put(self, request, path, pk=None):
 
             config_digest = config_layer.get("digest")
             try:
+                blobs_in_latest_version = repository.latest_version().content.values_list("pk")
```

Comment:
these 2 lines are repeated twice, you can re-use them and have just in one place, for example move them up right after config_digest line